docs/source/getting_started/getting_started.md

```diff
@@ -147,27 +164,38 @@ As a result, we will get something like:
 
 In the request, we ask the system to include the properties of the returned documents. We can use this data to implement a 'more like this' section.
 
-We also have a 'score' field which represents how well the documents match the user's interests. The higher the number, the better the documents match.
+We also have a `score` field which represents how well the documents match the user's interests. The higher the number, the better the documents match. It should be noted that the scores only have meaning in relation to other
+scores from the same requests.
+
+`snippet_id` is the id of the snippet which was found. The `document_id` in the `snippet_id` is the id of the document associated with the snippet. If you do not use ingestion options like `split` and in turn only have
+one snippet per document then you can always use `snippet_id.document_id` and ignore the rest. In many places
+both a the full snippet id object or a document id string can be used. If documents which have multiple snippets are ingested it's highly recommended to always use the full snippet id.
 
 # Search
 
-Depending on the use-case searching for documents can be achieved as a search for documents _similar_ to a given document or as a _free-text search_. Both variants can then be run as a anonymous search or a search that is personalized. Personalization comes in two fashions, with a _user-id_ or by providing a interaction _history_.
+Depending on the use-case searching for documents can be achieved as a search for documents _similar_ to a given snippet/document or as a _free-text search_. Both variants can then be run as a anonymous search or a search that is personalized. Personalization comes in two fashions, with a _user-id_ or by providing a interaction _history_.
 
 ## Similar documents
 
-In this search variant only a _document id_ must be provided to the [`/semantic_search`](https://docs.xayn.com/front_office.html#tag/front-office/operation/getSimilarDocuments) endpoint.
+In this search variant either a _document id_ string or a _snippet id_ object must be provided to the [`/semantic_search`](https://docs.xayn.com/front_office.html#tag/front-office/operation/getSimilarDocuments) endpoint.
```

Comment:
We already showed what a document and snippet ids are. I wouldn´t specify that they are string and objects.
```suggestion
In this search variant either a _document id_ or a _snippet id_ must be provided to the [`/semantic_search`](https://docs.xayn.com/front_office.html#tag/front-office/operation/getSimilarDocuments) endpoint.
```